services/traefik/docker-compose.local.yml

```diff
@@ -56,9 +50,6 @@ services:
         - traefik.http.routers.traefik_http_to_https.entrypoints=traefik
         - traefik.http.routers.traefik_http_to_https.middlewares=traefik_force_https
 
-        # traefik api
-        - traefik.http.routers.api.rule=PathPrefix(`/dashboard`) || PathPrefix(`/api`)
-        - traefik.http.routers.api.entrypoints=traefik
```

Comment:
I think there is a misunderstanding here.
this is the external traefik dashboard. this one should be kept. please revert